src/scout_apm/instruments/elasticsearch.py

```diff
@@ -84,11 +84,14 @@ def instrument_client(self):
 
         for method_str in self.__class__.CLIENT_METHODS:
             try:
-                code_str = """
+                code_str = """\
 @monkeypatch_method(Elasticsearch)
 def {method_str}(original, self, *args, **kwargs):
     tr = TrackedRequest.instance()
-    index = kwargs.get('index', 'Unknown').title()
+    index = kwargs.get('index', 'Unknown')
+    if isinstance(index, (list, tuple)):
+        index = ','.join(index)
```

Comment:
So with an array, we could end up with a name like `posts,articles` or something? This name is used as the layer name in our UI, so we probably want to figure out a better way to handle multiple strings from the array here.